### ISSUE TITLE
[0.81] fix(a11y): `aria-hidden` support for `Text`, non-editable `TextInput` and `Image`

### ISSUE THREAD

**Issue by mdjastrzebski** [reactwg/react-native-releases#1100]
### Target Branch

0.81

### Link to commit or PR to be picked

https://github.com/facebook/react-native/pull/53364

### Description

This address the missing support non-`View` key controls: `Text`, `TextInput`, `Image`. This is especially needed in our case as we use React Strict DOM, which exposes only ARIA a11y props.

**Comment by mdjastrzebski** [comment]
No worries, it's already available in 0.82.
